pyproject.toml

```diff
@@ -1,3 +1,8 @@
+# PEP-518 https://peps.python.org/pep-0518/
+[build-system]
+# Minimum requirements for the build system to execute.
+requires = ["setuptools>=38.0", "wheel"]  # PEP 508 specifications.
+
```

Comment:
This is something I just come across when I work on this PR, no related to the PR particularly.

Reply:
We've got `"setuptools>=38.0"` and `wheel` requirements in various other places in the repo (just do a Ctrl+F to see them). Does this mean we can remove them from there?

If yes then that would be nice but a bit of a bigger change, so probably worth doing in a separate PR.

Reply:
good point, created a new PR and will revert this soon.